TimeAndDate.Services/DataTypes/Time/TADDateTime.cs

```diff
@@ -0,0 +1,175 @@
+
+using System;
+using System.Xml;
+using TimeAndDate.Services.Common;
+using System.Globalization;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TimeAndDate.Services.DataTypes.Time
+{
+    public class TADDateTime
+    {
+        /// <value>
+        /// Year.
+        /// </value>
+        public Int32 Year { get; set; }
+
+        /// <value>
+        /// Month.
+        /// </value>
+        public Int32 Month { get; set; }
+
+        /// <value>
+        /// Day.
+        /// </value>
+        public Int32 Day { get; set; }
+
+        /// <value>
+        /// Hour.
+        /// </value>
+        public Int32 Hour { get; set; }
+
+        /// <value>
+        /// Minute.
+        /// </value>
+        public Int32 Minute { get; set; }
+
+        /// <value>
+        /// Second.
+        /// </value>
+        public Int32 Second { get; set; }
+
+        public TADDateTime() { }
+
+        public TADDateTime(Int32 year, int month, int day, int hour, int minute, int second)
+        {
+            Year = year;
+            Month = month;
+            Day = day;
+            Hour = hour;
+            Minute = minute;
+            Second = second;
+        }
+        public TADDateTime(Int32 year, int month, int day)
+        {
+            Year = year;
+            Month = month;
+            Day = day;
+        }
+
+        public TADDateTime(DateTimeOffset date)
+        {
+            Year = date.Year;
+            Month = date.Month;
+            Day = date.Day;
+            Hour = date.Hour;
+            Minute = date.Minute;
+            Second = date.Second;
+        }
+
+        public TADDateTime(DateTime date)
+        {
+            Year = date.Year;
+            Month = date.Month;
+            Day = date.Day;
+            Hour = date.Hour;
+            Minute = date.Minute;
+            Second = date.Second;
+        }
+
+        public DateTimeOffset ToStd()
```

Comment:
I think you can use explicit operator conversion here
```
public static explicit operator DateTimeOffset(TADDateTime date)
{
        return new DateTimeOffset(date.Year, etc...)
}
 ```

So people can just cast it
```
var taddate = new TADDateTime(etc, etc);
var offset = (DateTimeOffset)taddate;
```